src/converters/app-converter.ts

```diff
@@ -247,11 +241,11 @@ export class StackConverter extends ArtifactConverter {
 
     /** @internal */
     asOutputValue<T>(v: T): T {
-        const value = this.stackComponent.stack.resolve(v);
+        const value = this.cdkStack.resolve(v);
         return this.processIntrinsics(value) as T;
     }
 
-    private processIntrinsics(obj: any): any {
+    public processIntrinsics(obj: any): any {
```

Comment:
This was necessary? I guess we can mark the entire class as implementation-level class so this is not the user facing API.

Reply:
We currently do not export this class to users.